src/site/content/en/blog/learn-to-bundle/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Learn to bundle
+subhead: |
+    Learn to bundle your website.
```

Comment:
I'd suggest punching up this subhead a bit if possible. Maybe mentioning why someone would want to bundle their site.